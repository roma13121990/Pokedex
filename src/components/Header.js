import React, {Component} from 'react';
import {Link} from "react-router-dom";

const styles = {

    'background': '#ffffff',
    'padding': 16,
    'fontSize': '1.5em',
    'color': '#515673',
    'textAlign': 'left',
    'position': 'fixed',
    'left':0,
    'right':0,
    'top': 0,
    'zIndex': 99

};
const button = {
    "float": "right",
    "textDecoration": "none",
    "fontSize": "1rem",
    "color": "#ffffff",
    "background": "#f66",
    "padding": "5px 10px",
    "borderRadius": "19px"
}
export default class Header extends Component {
    render() {
        return (<div style={styles}>Pokedex
                <Link style={button} to='/MyPokemonList'>My Pokemon</Link>
            </div>

        )
    }
}