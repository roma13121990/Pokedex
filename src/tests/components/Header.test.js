import React from 'react';
import renderer from 'react-test-renderer';
import Header from "../../components/Header";
import {MemoryRouter} from "react-router-dom";

it('should render header correctly', () => {
    const tree = renderer
        .create(<MemoryRouter>
            <Header />
        </MemoryRouter>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});