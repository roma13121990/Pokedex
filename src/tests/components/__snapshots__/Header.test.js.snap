// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render header correctly 1`] = `
<div
  style={
    Object {
      "background": "#ffffff",
      "color": "#515673",
      "fontSize": "1.5em",
      "left": 0,
      "padding": 16,
      "position": "fixed",
      "right": 0,
      "textAlign": "left",
      "top": 0,
      "zIndex": 99,
    }
  }
>
  Pokedex
  <a
    href="/MyPokemonList"
    onClick={[Function]}
    style={
      Object {
        "background": "#f66",
        "borderRadius": "19px",
        "color": "#ffffff",
        "float": "right",
        "fontSize": "1rem",
        "padding": "5px 10px",
        "textDecoration": "none",
      }
    }
  >
    My Pokemon
  </a>
</div>
`;
